CHANGELOG.md

```diff
@@ -9,8 +9,17 @@ Please note that the date associated with a release is the date the code
 was committed to the `main` branch. This is not necessarily the date that
 the code was deployed.
 
+
+
 ## [Unreleased]
 
+### Changes Made - 2023-09-22
```

Comment:
I would suggest making the headings consistent with the rest of the Changelog as per below. So use ### Added and/or ### Removed, when describing your changes. Then note what you added or removed in your code.